dcpy/test/lifecycle/package/test_column_validation.py

```diff
@@ -58,24 +58,26 @@ def bbl(boro_code, block, lot):
     "bbl": DCPFakes.bbl,
     "uid": lambda: str(uuid.UUID(int=rd.getrandbits(128), version=4)),
     "text": faker.pystr,
+    "integer": lambda: str(faker.pyint()),
+    "decimal": lambda: str(faker.pyfloat()),
     "latitude": lambda: str(faker.latitude()),
     "longitude": lambda: str(faker.longitude()),
 }
 
 
-def _fake_row(columns: list[models.Column]):
+def _fake_row(columns: list[md.DatasetColumn]):
     row = {}
 
     found_bbl_parts = {}
     bbl_parts = {"boro_code", "block", "lot"}
     found_bbl_name = ""
     for c in columns:
         if c.data_type == "bbl":
-            found_bbl_name = c.name
+            found_bbl_name = c.name or ""
         elif c.values:
-            row[c.name] = random.choice(c.values)[0]
+            row[c.name] = random.choice(c.values).value
         else:
-            val = fakes[c.data_type]()
+            val = fakes[c.data_type or ""]()
```

Comment:
appeasing type checker. `data_type` is nullable because of how inheritance works in the DatasetColumn. I'm not entirely happy with that...